.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Just a reminder that text in these brackets is not visible in the rendered output
+You also do *not* need to use this template. Its just a suggestion :)
+-->
+
+
+
+## Changes
+<!-- Please describe what this Pull request adds or changes. Possibly with  images -->
+
+## Todo before merging
+<!-- Add custom todos if deemed necessary to give others an overview on how far this PR is progressing -->
+
+<!-- Did you add UI text? 
+No? Just check or remove the box
+Yes? Make sure you remove hardcoded strings with translations -->
+- [ ] Translations
+<!--
+Did you add settings to the settings screen?
+No? Just check or remove the box 
+Yes? Make sure they are resettable using the button on the respective page! -->
+- [ ] Reset Settings
+
+
+## Related Issues
+<!-- List issues which may or may not be closed when this PR merges -->
```

Comment:
~~We could add a section of "related issues or pull requests" here,~~ didn't see this at first, whoops  
prompt the user to use closing keywords, like

- closes #xyz
- fixes #abc
- follow up to #pr-name

Maybe there's even an input element for linking issues?

Reply:
I dont think PRs can be yml, so no input element. Besides PRs dont need much of a structure in the first place bth xd